tests/test_query_planner.py

```diff
@@ -489,30 +343,29 @@ async def test_unbound_unconnected_node(caplog):
 async def test_invalid_two_disconnected_components(caplog):
```

Comment:
This name also should be changed to valid.

Reply:
77c3964a535622072698e8be3c438ec4ab04dae6